docs/source/api/index.md

```diff
@@ -46,24 +53,192 @@ More coming soon...
+
+The locations checked are the following:
+
+1. Current working directory, {func}`os.getcwd`
+1. Location indicated by {envvar}`ARVIZ_DATA` environment variable
+1. The third and last location checked is OS dependent:
+   * On Linux: `$XDG_CONFIG_HOME/arviz` if exists, otherwise `~/.config/arviz/`
+   * Elsewhere: `~/.arviz/`
+
+:::{dropdown} Example `arvizrc` file
+:name: arvizrc
+:open:
+
+```none
+data.index_origin : 1
+plot.backend : bokeh
+stats.ci_kind : hdi
+stats.ci_prob : 0.95
+```
+
+All available keys are listed below. The `arvizrc` file can have any subset of the keys,
+it isn't necessary to include them all. For those keys without a user defined default,
+the library one is used.
+:::
+
+### Context manager
+A context manager is also available to temporarily change the default settings.
 
 ```{eval-rst}
 .. autosummary::
    :toctree: generated/
 
-   arviz_base.convert_to_dataset
-   arviz_base.dict_to_dataset
-   arviz_base.generate_dims_coords
-   arviz_base.make_attrs
-   arviz_base.ndarray_to_dataarray
+   arviz_base.rc_context
+```
+
+## rcParams
+Below, all keys available within `rcParams` are listed, along with their library default.
+
+Keys can be accessed or modified via ``arviz_base.rcParams[key]``, for example,
+``arviz_base.rcParams["data.sample_dims"]``.
+
+:::{important}
+These defaults are subject to change. If you rely on a specific default value,
+you should create an {ref}`arvizrc <arvizrc>` file with the key-value pairs you rely on.
+
+The goal of the ArviZ team is to try and adapt to best practices as they evolve,
+which sometimes requires updating default values, for example to use new algorithms.
+:::
+
+### data
+
+
+```{eval-rst}
+.. py:data:: data.http_protocol
+    :type: str
+    :value: "https"
+
+    Protocol for loading remote datasets. Can be "https" or "http".
+
+.. py:data:: data.index_origin
+    :type: int
+    :value: 0
+
+    Index origin. By default, ArviZ adds coordinate values to all dimensions.
+    If no coordinate values were provided, ArviZ generates integer indices
+    as coordinate values starting at `index_origin`.
+
+.. py:data:: data.log_likelihood
+    :type: bool
+    :value: True
+
+    Whether to save pointwise log likelihood values.
+
+.. py:data:: data.sample_dims
+    :type: list
+    :value: ["chain", "draw"]
+
+    What the sampling dimensions are named. These are the dimensions that will be
+    reduced by default when computing or plotting, therefore, they should be always present.
+
+.. py:data:: data.save_warmup
+    :type: bool
+    :value: False
+
+    Whether to save warmup iterations.
+```
+
+### stats
+
+```{eval-rst}
+.. py:data:: stats.module
+    :type: str
+    :value: "arviz_stats.base"
+
+    Preferred module for stats computations
+
+.. py:data:: stats.ci_kind
+    :type: str
+    :value: "eti"
+
+    Type of credible interval to compute by default, one of "eti" or "hdi".
+
+.. py:data:: stats.ci_prob
+    :type: float
+    :value: 0.83
```

Comment:
When did we switch to 0.83 vs 0.94?

Reply:
opened an issue #4 to discuss all rcParams changes and updates